### ISSUE TITLE
Wrong link in /docs/checks.md file

### ISSUE THREAD

**Issue by AleX04Nov** [ossf/scorecard#4362]
**Describe the bug**
There is a wrong link in /docs/checks.md file: 
https://github.com/ossf/scorecard/blob/6d9b2e4f78f7389961cb2e80851b4e901cf635bc/docs/checks.md (the latest commit as of the day of posting this bug)
In the SBOM section, it leads to https://github.com/ossf/SBOM-everywhere/blob/main/reference/SBOM_naming.md#consistent-naming-conventions ; however, this file does not exist. The correct name is 
https://github.com/ossf/SBOM-everywhere/blob/main/reference/sbom_naming.md#consistent-naming-conventions

**Reproduction steps**
Steps to reproduce the behaviour:
1. Open the /docs/checks.md file
2. Navigate to the SBOM section
3. Click this link: https://github.com/ossf/SBOM-everywhere/blob/main/reference/SBOM_naming.md#consistent-naming-conventions
4. Observe the 404 error.

**Expected behaviour**
The expected behaviour is to see a correct document under this link:
https://github.com/ossf/SBOM-everywhere/blob/main/reference/sbom_naming.md#consistent-naming-conventions
TLDR: The filename should be in lowercase.

**Comment by AleX04Nov** [comment]
There is already an opened pull request resolving this issue:

https://github.com/ossf/scorecard/pull/4334
